src/test/java/pages/ProductsPage.java

```diff
@@ -13,6 +13,11 @@ public class ProductsPage extends BasePage {
     private final static By ITEM_PRICE = By.className("inventory_item_price");
     private final static By ITEM_DESCRIPTION = By.className("inventory_item_desc");
 
+    public final static String DESCRIPTION_PRODUCT = "Rib snap infant onesie for the junior automation engineer in development. Reinforced 3-snap bottom closure, two-needle hemmed sleeved and bottom won't unravel.";

```

Comment:
That's not  correct. Page object classes should not store any test data. Test data should be stored in test classes!